ddocs/medic-client/views/contacts_by_freetext/map.js

```diff
@@ -1,5 +1,16 @@
 function(doc) {
-  var skip = [ '_id', '_rev', 'type', 'refid' ];
+  var skip = [ '_id', '_rev', 'type', 'refid', 'geolocation' ];
+
+  var usedKeys = [];
+  var emitMaybe = function(key, value) {
+    if (usedKeys.indexOf(key) === -1 && // Not already used
```

Comment:
Or for bonus points, maybe drop the use of the array for the key and just use the string. I don't know what sort of overhead having a single element array would add over just a plain string but it might be worthwhile.